rd_ui/app/views/dashboard.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
                 </div>
             </div>
 
-            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
```

Comment:
Using ng-hide, the code inserted into the widget was not present in the page, so the functionality fails. For example, you created a widget with a javascript inside as text. The text shuold be present in the page but not displayed in the widget. Well, with ng-hide, the text was hidden to, NOT present in the page, so the functionality fails. If you have any question, please let me know. Thanks.